### ISSUE TITLE
SUSHI Import Confirmation

### ISSUE THREAD

**Issue by nicksjunkyard** [coral-erm/coral#672]
Small imports process successfully but large reports give 500 error.  ProQuest, Springer and Oxford typically give the error.  No fatal errors are showing in the php log file.   The files are typically more than 3MB.
This page isn’t working right now
.edu can't currently handle this request.
HTTP ERROR 500

The log file notices look like this
[12-Nov-2020 11:10:54 America/New_York] PHP Notice:  Undefined variable: addWhere in E:\media.lib\coral\usage\admin\classes\domain\Title.php on line 92
[12-Nov-2020 11:10:54 America/New_York] PHP Notice:  Undefined index: Proprietary Identifier in E:\media.lib\coral\usage\uploadComplete.php on line 249
[12-Nov-2020 11:10:54 America/New_York] PHP Notice:  Undefined index: Proprietary Identifier in E:\media.lib\coral\usage\uploadComplete.php on line 249
[12-Nov-2020 11:10:54 America/New_York] PHP Notice:  Undefined index: DOI in E:\media.lib\coral\usage\uploadComplete.php on line 249

**Comment by andyp-uk** [comment]
Can we verify that you are using version 2020.09? Secondly, are you able to share with us the SUSHI config that you are using, and the date parameters for the import job?

**Comment by nicksjunkyard** [comment]
Thank you.  Yes, we installed the 2020.09 update.  Is it possible to send the API key and customer ID privately?  The date was just 2019-09-01 - 2019-09-30 for ProQuest TR_J1 counter 5.

**Comment by andyp-uk** [comment]
Yes—I see one of your colleagues has just emailed me. Am just about to send my reply.

**Comment by andyp-uk** [comment]
Just to confirm that I was able to import your ProQuest TR_J1 report for Sep 2019 on my system. Admittedly I am using a slightly different version (sirsidynix/coral) although I don't think there's a significant difference in the code. Have you verified the 2020.09 database upgrade completed successfully? Assuming it did, I wonder if there is some system-specific limit that is being exceeded. Might be worth us comparing our php configurations. I sent your colleague some additional info, which might help.
